packages/swap/src/handlers/__tests__/openSwapDepositChannel.test.ts

```diff
@@ -17,34 +17,33 @@ jest.mock('../../utils/screenAddress', () => ({
 jest.mock('axios');
 
 describe(openSwapDepositChannel, () => {
-  beforeAll(() => {
+  beforeAll(async () => {
     jest
       .useFakeTimers({ doNotFake: ['nextTick', 'setImmediate'] })
       .setSystemTime(new Date('2022-01-01'));
+
+    await prisma.chainTracking.create({
+      data: {
+        chain: 'Ethereum',
+        height: BigInt('125'),
+        blockTrackedAt: new Date('2023-11-09T10:00:00.000Z'),
+      },
+    });
   });
 
-  it('gathers and inserts the deposit channel info', async () => {
-    jest.mocked(axios.post).mockResolvedValueOnce({ data: environment() });
+  beforeEach(async () => {
+    await prisma.$queryRaw`TRUNCATE TABLE "SwapDepositChannel" CASCADE`;
+  });
 
+  it('creates channel and stores it in the database', async () => {
+    jest.mocked(axios.post).mockResolvedValueOnce({ data: environment() });
     jest.mocked(broker.requestSwapDepositAddress).mockResolvedValueOnce({
       sourceChainExpiryBlock: BigInt('1000'),
       address: 'address',
       channelId: BigInt('888'),
       issuedBlock: 123,
     });
 
-    prisma.swapDepositChannel.upsert = jest.fn().mockResolvedValueOnce({
```

Comment:
i changed this to use the database because this approach would not have caught this bug (because it does not invoke the db code)